### ISSUE TITLE
Replot Loss After Train

### ISSUE THREAD

**Issue by Songloading** [junyanz/pytorch-CycleGAN-and-pix2pix#991]
Hi, I'm just wondering if there is anyway I can replot the loss from the txt file after training finished?

**Comment by junyanz** [comment]
It is currently not supported. If you use visdom, you should be able to save the plot form visdom console.
